.travis.yml

```diff
@@ -44,3 +44,13 @@ script:
   - coverage report -m
 after_success:
   - coveralls
+# Configure travis to deploy to the pypi server when a new
+# version is tagged on master
+deploy:
+  provider: pypi
+  user: "trackstand"
```

Comment:
Hopefully/presumably the deploy will be triggered and run successfully if others create a new release (i.e. me!), despite it being in your name. I'm suspect it will work fine.